src/control/conf.c

```diff
@@ -327,6 +327,14 @@ const char *dt_conf_get_string_const(const char *name)
   return dt_conf_get_var(name);
 }
 
+gboolean dt_conf_key_valid(const char *name)
```

Comment:
The conf key is valid but with an empty value. I would rename this routine : `dt_conf_key_not_empty()` or something like that.

Reply:
Right - i thought about the name before, i took your suggestion now. Indeed much better